lib/oli_web/templates/open_and_free/show.html.eex

```diff
@@ -21,7 +21,7 @@
       </tr>
       <tr>
         <td><strong>URL:</strong></td>
-        <td><%= external_link Routes.page_delivery_url(@conn, :index, @section.slug), to: Routes.page_delivery_url(@conn, :index, @section.slug) %></td>
+        <td><%= url(~p"/sections/#{@section.slug}") %></td>
```

Comment:
This page_delivery route no longer exists so I updated it to the new route, even though this `show` view likely isn't being used anymore anyway